### PR TITLE
[nginx_ingress_controller] Migrate to format_version v3

### PR DESCRIPTION
**Pull request by tetianakravchenko**
## Proposed commit message

- WHAT: Migrate to format_version v3 and fix validation errors, fix validation errors:
```
1. file "/Users/tetianakravchenko/go/src/github.com/elastic/integrations/build/packages/nginx_ingress_controller-1.8.3.zip/data_stream/access/fields/fields.yml" is invalid: field 0.fields.0.type: 0.fields.0.type must be one of the following: "aggregate_metric_double", "alias", "histogram", "constant_keyword", "text", "match_only_text", "keyword", "long", "integer", "short", "byte", "double", "float", "half_float", "scaled_float", "date", "date_nanos", "boolean", "binary", "integer_range", "float_range", "long_range", "double_range", "date_range", "ip_range", "group", "geo_point", "object", "ip", "nested", "flattened", "wildcard", "version", "unsigned_long"
   2. file "/Users/tetianakravchenko/go/src/github.com/elastic/integrations/build/packages/nginx_ingress_controller-1.8.3.zip/manifest.yml" is invalid: field (root): Additional property release is not allowed
   3. file "/Users/tetianakravchenko/go/src/github.com/elastic/integrations/build/packages/nginx_ingress_controller-1.8.3.zip/manifest.yml" is invalid: field (root): Additional property license is not allowed
   4. file "/Users/tetianakravchenko/go/src/github.com/elastic/integrations/build/packages/nginx_ingress_controller-1.8.3.zip/manifest.yml" is invalid: field owner: type is required
   5. file "/Users/tetianakravchenko/go/src/github.com/elastic/integrations/build/packages/nginx_ingress_controller-1.8.3.zip/manifest.yml" is invalid: field conditions: Additional property kibana.version is not allowed
   6. field "host.ip" is defined multiple times for data stream "access", found in: /Users/tetianakravchenko/go/src/github.com/elastic/integrations/build/packages/nginx_ingress_controller-1.8.3.zip/data_stream/access/fields/agent.yml, /Users/tetianakravchenko/go/src/github.com/elastic/integrations/build/packages/nginx_ingress_controller-1.8.3.zip/data_stream/access/fields/ecs.yml
   7. references found in dashboard kibana/dashboard/nginx_ingress_controller-0b3dba40-f341-11ea-a3fd-1b45ec532bb3.json: nginx_ingress_controller-78ebd0d0-404a-11eb-b2e4-0f0dbcd50464 (search), nginx_ingress_controller-d20d4ea0-f342-11ea-a3fd-1b45ec532bb3 (search) (SVR00004)
   8. file "/Users/tetianakravchenko/go/src/github.com/elastic/integrations/build/packages/nginx_ingress_controller-1.8.3.zip/kibana/dashboard/nginx_ingress_controller-dfbc0840-f340-11ea-a3fd-1b45ec532bb3.json" is invalid: expected filter in dashboard: no filter found and at least one panel does not have a filter (SVR00002)
```
- WHY:  https://github.com/elastic/integrations/issues/10174

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ] 

## Related issues

- Closes https://github.com/elastic/integrations/issues/10174

## Screenshots
New integration version:
Screenshot 2024-06-28 at 13 09 58

Screenshot 2024-06-28 at 13 10 20

### CODE REVIEW

File: packages/nginx_ingress_controller/manifest.yml [hunk 1]
```diff
@@ -1,17 +1,16 @@
-format_version: 1.0.0
+format_version: 3.0.4
```
Comment (constanca-m) [inline packages/nginx_ingress_controller/manifest.yml:1]: Is there reason to leave it as `3.0.4` and not the latest version available for `format_version`?

I see [this](https://github.com/elastic/package-spec?tab=readme-ov-file#version-compatibility-between-packages-and-the-stack), but I am not sure if it is related:

> This would mean that it is compatible with packages using any spec version >= 2.0.0 and <3.1.0.

File: packages/nginx_ingress_controller/data_stream/access/fields/agent.yml [hunk 2]
```diff
@@ -121,10 +121,6 @@
         As hostname is not always unique, use values that are meaningful in your environment.
 
         Example: The current usage of `beat.name`.'
-    - name: ip
```
Comment (constanca-m) [inline packages/nginx_ingress_controller/data_stream/access/fields/agent.yml:124]: Why is this removed? I don't see that in the issue description. It holds no value?

Comment (tetianakravchenko) [reply packages/nginx_ingress_controller/data_stream/access/fields/agent.yml:124]: this field was duplicated:
```
field "host.ip" is defined multiple times for data stream "access", found in: /Users/tetianakravchenko/go/src/github.com/elastic/integrations/build/packages/nginx_ingress_controller-1.8.3.zip/data_stream/access/fields/agent.yml, /Users/tetianakravchenko/go/src/github.com/elastic/integrations/build/packages/nginx_ingress_controller-1.8.3.zip/data_stream/access/fields/ecs.yml
```

Comment (tetianakravchenko) [reply packages/nginx_ingress_controller/data_stream/access/fields/agent.yml:124]: I've updated the description with the list of validation errors

File: packages/nginx_ingress_controller/manifest.yml (hunk 1 above)
Comment (constanca-m) [reply packages/nginx_ingress_controller/manifest.yml:1]: Yes, it seems like a mess, good idea to change gradually then

**Review by constanca-m — APPROVED** [review]

**Comment by tetianakravchenko** [comment]
@constanca-m @gizas 
the error:
> Error: can't install the package: could not install package; API status code = 500; response body = {"statusCode":500,"error":"Internal Server Error","message":"Migration function for version 7.11.0 threw an error"}

since I've updated dashboards, I've also bumped Kibana version
